src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -137,6 +161,15 @@ public static boolean deleteFolder(final File file)
         return false;
     }
 
+    public static boolean deleteFile(final File file)
```

Comment:
oh, right, remembered! is used to delete coreprotect database when it reachs a certain weight.